Modix.Bot/Modules/DocumentationModule.cs

```diff
@@ -24,36 +27,49 @@ public DocumentationModule(DocumentationService documentationService)
             [Summary("The term to search for in the documentation.")]
                 string term)
         {
+
+            Regex reg = new Regex("^[0-9A-Za-z.<>]+$");
+            foreach(char c in term)
+            {
+                if(!reg.IsMatch(c.ToString()))
+                {
+                    await ReplyAsync($" ' {c} ' character is not allowed in the search, Please try again.");
+                    return;
+                }
+            }
+
             var response = await DocumentationService.GetDocumentationResultsAsync(term);
 
             if (response.Count == 0)
             {
                 await ReplyAsync("Could not find documentation for your requested term.");
                 return;
             }
-            
+
             var embedCount = 0;
 
-            foreach (var res in response.Results.Take(3).OrderBy(x => x.DisplayName))
+            var stringBuild = new StringBuilder();
+
+            foreach (var res in response.Results.Take(3))
             {
                 embedCount++;
-
-                var builder = new EmbedBuilder()
-                    .WithColor(new Color(46, 204, 113))
-                    .WithTitle($"{res.ItemKind}: {res.DisplayName}")
-                    .WithUrl(res.Url)
-                    .WithDescription(res.Description);
+                stringBuild.Append($"**\u276F [{res.ItemKind}: {res.DisplayName}]({res.Url})**{Environment.NewLine}");
```

Comment:
You could do `stringBuild.AppendLine(...);` for this and the following two lines instead of putting `Environment.NewLine` at the end of each line.